doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -164,136 +162,142 @@ Enable partition mode in ACRN hypervisor
       Refer to the :ref:`getting-started-building` for more information on how
       to install all the ACRN build dependencies.
 
+#. Prepare VM configurations for UP2 partition mode
+
+   The BOARD specific VM configurations should be under the folder:
+   ``hypervisor/arch/x86/configs/$(CONFIG_BOARD)/``.
+
+   For UP2 board, we can simply copy configurations of apl-mrb to the up2 folder:
+
+   .. code-block:: none
+
+     $ cp hypervisor/arch/x86/configs/apl-mrb/* hypervisor/arch/x86/configs/up2/
```

Comment:
... this works as written since there are no subdirectories in ``apl-mrb`` so I'll leave it alone

Reply:
We should at least create the up2 folder beforehand. That command will fail otherwise (this was in fact my primary motivation for suggesting this change)